### PR TITLE
GH-2339: As a developer I want to replace some @n4jsd packages with @types packages (part 1)

### LINKED ISSUE #2339: As a developer I want to replace some @n4jsd packages with @types packages
In n4js-libs and other n4js projects we use @n4jsd projects to have typed access to npm packages. Since we have first support of @types projects already this userstory is about migrating some of @n4jsd dependencies to @types ones.

### PR DESCRIPTION
**Pull request by mmews-n4**
closes #2339

### CODE REVIEW

**Comment by mmews-n4** [comment]
This is an intermediate PR to make the current changes available on master.

File: plugins/org.eclipse.n4js/src/org/eclipse/n4js/utils/ProjectDescriptionLoader.java [hunk 1]
```diff
@@ -180,6 +180,13 @@ private void adjustMainPath(FileURI location, JSONDocument packageJSON) {
 		JSONObject contentCasted = (JSONObject) content;
 		NameValuePair mainProperty = JSONModelUtils.getNameValuePair(contentCasted, MAIN.name).orElse(null);
 
+		// FIXME: move or delete following patch!
+		// if (mainProperty == null) {
+		// mainProperty = JSONModelUtils.addProperty(contentCasted, MAIN.name, mainValue);
+		// } else {
+		// mainProperty.setValue(mainValue);
+		// }
```
Comment (mor-n4) [inline plugins/org.eclipse.n4js/src/org/eclipse/n4js/utils/ProjectDescriptionLoader.java:188]: can this be deleted?

**Review by mor-n4 — APPROVED** [review]
:+1:

**Comment by mmews-n4** [comment]
Thank you! Build is green... merging.

**Pull request merged**